api/api/controllers/search_controller.py

```diff
@@ -9,7 +9,7 @@
 from django.conf import settings
 from django.core.cache import cache
 
-from elasticsearch.exceptions import NotFoundError, RequestError
+from elasticsearch.exceptions import NotFoundError, BadRequestError
```

Comment:
I believe the lint error we're seeing is about the order of the imports. Imports are sorted alphabetically so placing BadRequestError first should fix it.

Reply:
Understood @dhruvkb, will change in a while.